src/main/java/slimeknights/tconstruct/smeltery/item/TankItem.java

```diff
@@ -0,0 +1,67 @@
+package slimeknights.tconstruct.smeltery.item;
+
+import net.minecraft.block.Block;
+import net.minecraft.client.util.ITooltipFlag;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.BlockItem;
+import net.minecraft.item.IItemPropertyGetter;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.world.World;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+import net.minecraftforge.fluids.FluidStack;
+import net.minecraftforge.fluids.capability.templates.FluidTank;
+import slimeknights.mantle.item.BlockTooltipItem;
+import slimeknights.tconstruct.smeltery.tileentity.TankTileEntity;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+public class TankItem extends BlockTooltipItem {
+
+  public TankItem(Block blockIn, Properties builder) {
+    super(blockIn, builder);
+    this.addPropertyOverride(new ResourceLocation("tconstruct","amount"), (stack, world, entity) -> {
+      System.out.println("test1");
```

Comment:
This looks wrong.

I actually suggest just removing the item amount override stuff for now, it needs the retexturable model to get the fluid textures right.

Reply:
did.